README.md

```diff
@@ -1,3 +1,9 @@
-# Northcoders News API
+# News API
+
+When using this respository, create two .env files:
+.env.development
+.env.test
+
+this will allow you to succesfully connect to the database's locally.
+
```

Comment:
You will need to specify what the databases are called here too not just the files needed. Please add this before merging 👍